docs/bigquery/snippets.py

```diff
@@ -536,6 +656,41 @@ def test_update_table_cmek(client, to_delete):
     # [END bigquery_update_table_cmek]
 
 
+def test_browse_table_data(client, to_delete, capsys):
+    """Retreive selected row data from a table."""
+
+    # [START bigquery_browse_table]
```

Comment:
I see that "max_results" is used for all `list_rows` calls in this snippet. Could we add one without that so that we can explain that it's an iterator over all pages of results?

Reply:
Added in c7d849b